backend/repository/data/client/baz_client_update.json

```diff
@@ -0,0 +1,33 @@
+{
+    "_comment_": "JSON for updating a tchannel client.",
```

Comment:
Thought about it: JSON schema reserves string "description", which could be a key and is not good for the general purpose of a JSON. `_description_` or `_comment_` `__description__` or `__comment__` all seem good to me (they should be the first one lexicographically).